open-spatial-lab/dextra
# Density map

## Overview
Add density map to the story map .

### Demo

<img width="908" alt="image" src="https://github.com/open-spatial-lab/dextra/assets/102711675/0777ea41-d563-4289-86b4-c1a211418f0a">

### Note
The plot uses states.json and nations.json as boundaries, Walmart.tsv to make dots and density. 

## Testing Instructions
- How to test this PR: Run "pnpm dev:components" in terminal, and goes to "http://localhost:6006". 
- Expected output: You should see "Density map" in the left menu.Click it you should see the visualization shown in demo.
